pkg/liquidity-mining/test/AuthorizerAdaptorEntrypoint.test.ts

```diff
@@ -151,6 +152,10 @@ describe('AuthorizerAdaptorEntrypoint', () => {
       });
```

Comment:
I'd wrap

```
      it('performs the expected function call', async () => {
        const value = await entrypoint.connect(grantee).callStatic.performAction(target, calldata);
        expect(value).to.be.eq(expectedResult);
      });
      it('sends value to target contract correctly', async () => {
        const value = await entrypoint
          .connect(grantee)
          .callStatic.performAction(payableTarget, payableCalldata, { value: payment });
        expect(value).to.be.eq(payableExpectedResult);
      });
      it('rejects direct calls from the adaptor', async () => {
        // The authorizer will reject calls that are not initiated in the adaptor entrypoint.
        await expect(adaptor.connect(grantee).performAction(target, calldata)).to.be.revertedWith('SENDER_NOT_ALLOWED');
      });
```

into a `itDoesXCorrectly` function to reduce some repetition here.